instances.go

```diff
@@ -74,7 +75,7 @@ type InstanceAlert struct {
 // InstanceBackup represents backup settings for an instance
 type InstanceBackup struct {
 	Enabled  bool `json:"enabled,omitempty"`
-	Schedule struct {
+	Schedule *struct {
```

Comment:
This is _technically_ a breaking chance but it seems like it never worked correctly in the first place. We should include this in our release notes just in case.

Reply:
doesn't actually change anything sadly, removing it